### PR TITLE
Added Onju Voice as a supported media player

### PR DESCRIPTION
**Pull request by tetele**
Add the Onju Voice (https://github.com/justLV/onju-voice) as a media player/voice assistant

Source of the config: https://gist.github.com/tetele/75815c53cfd2a02412ef153f9c2d24e2 tested and improved for a few days

### CODE REVIEW

File: media-player/onju-voice.yaml [hunk 1]
```diff
@@ -0,0 +1,482 @@
...
+    entity_category: config
+    initial_value: 1.25
+    min_value: 0.25
+    max_value: 5
+    step: 0.25
+    optimistic: true
+    on_value:
+      then:
+        - lambda: !lambda |-
+            id(thresh_percent) = 0.01 * x;
```
Comment (jesserockz) [inline media-player/onju-voice.yaml:341]: Does this need to be exposed by default?
Perhaps add `disabled_by_default: true`

File: media-player/onju-voice.yaml [hunk 2]
```diff
@@ -0,0 +1,482 @@
...
+          then:
+            - voice_assistant.start_continuous:
+    on_release:
+      - if:
+          condition:
+            and:
+              - switch.is_on: use_wake_word
+              - binary_sensor.is_off: mute_switch
+          then:
+            - voice_assistant.stop:
```
Comment (jesserockz) [inline media-player/onju-voice.yaml:267]: ESPHome 2023.10.5 has some new triggers on the voice assistant that will provide a better experience for this. See https://github.com/esphome/firmware/pull/86

**Review by jesserockz — CHANGES_REQUESTED** [review]
Thanks for the hard work of getting the different components working =)

A few suggestions below

File: media-player/onju-voice.yaml (hunk 1 above)
Comment (tetele) [reply media-player/onju-voice.yaml:341]: Good point

**Review by jesserockz — APPROVED** [review]

**Comment by balloob** [comment]
Shouldn't this have been added to the `voice-assistant` folder?

**Comment by tetele** [comment]
From.what i've noticed, the configs here all have `voice_assistant` and the configs in `voice-assistant` are all based on `speaker`.

**Comment by balloob** [comment]
Some have had media player because that project came first. ESP32 S3 Box and Onju are both dedicated voice assistants and should live there.

**Comment by tetele** [comment]
Tbh, i see the Onju, just like the Nest Mini it sits in, as a great media player. But i'm happy to move it if you think it's best.

**Pull request merged**